pages/_app.tsx

```diff
@@ -1,6 +1,7 @@
+import type { AppProps } from "next/dist/next-server/lib/router/router";
```

Comment:
```suggestion
import type { AppProps } from "next/dist/next-server/lib/router/router";
```
We used just "next/app" as path

Reply:
thanks ✌️